js/views/PageNav.js

```diff
@@ -138,7 +137,10 @@ export default class PageNav extends View {
     });
 
     this.$addressBar = this.$('.js-addressBar');
+    this.$navList = this.$('.js-navList');
+    this.$popMenus = this.$('.js-navPopMenu');
 
+    getBody().on('click', (e) => { this.onBodyClick(e); });
```

Comment:
No reason on the wrapping function, it's a leftover from earlier when it had more code in it. 

Good point on doing it in render.

I originally had it on document, then I thought it seemed too much to add a getDoc to selectors.js. I'll put that back in. 

Thanks for the reminder on remove.